src/main/java/com/checkout/forex/RatesQueryFilter.java

```diff
@@ -0,0 +1,27 @@
+package com.checkout.forex;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NonNull;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class RatesQueryFilter {
```

Comment:
I wondered if we need the NoArgConstructor, but all the properties are required, so I think this is the right approach

Reply:
At first I used the NoArgConstructor annotation but it suggested to remove it because it was unnecessary because of that same reason